packages/legend-query-builder/src/components/QueryBuilderResultPanel.tsx

```diff
@@ -853,6 +870,47 @@ const QueryBuilderGridResult = observer(
       });
     };
 
+    const getSelectedCells = (
+      api: DataGridApi<unknown>,
+    ): QueryBuilderTDSResultCellData[] => {
```

Comment:
Have you tried using apis mentioned here to copy selected cells instead of writing custom function: https://www.ag-grid.com/javascript-data-grid/clipboard/#copying-via-the-api
I